config/crd/v1/bases/velero.io_backuprepositories.yaml

```diff
@@ -104,6 +104,34 @@ spec:
                 - Ready
                 - NotReady
                 type: string
+              recentMaintenanceStatus:
```

Comment:
nit:  It's a little repetitive to call it `status. recentMaintenanceStatus`, what if make it `status.recentMaintenance`.

Reply:
Done, changed it to `status.recentMaintenance`